app/util/bitbucket/populate_db.sh

```diff
@@ -0,0 +1,165 @@
+#!/bin/bash
+
+###################    Check if NFS exists        ###################
+pgrep nfsd > /dev/null && echo "NFS found" || echo "NFS was not found. Please check if you use correct machine"; exit 1
```

Comment:
Let's improve the wording a bit:
"NFS process was not found. This script is intended to run only on the Bitbucket NFS Server machine."